src/api/views.py

```diff
@@ -37,6 +37,72 @@ def get(self, request):
                 status=501,
             )
 
+# View 3
+class VictimGenders(generics.ListCreateAPIView):
+    def get(self, request, *args, **kwargs):
+        c_id = request.GET.get("community_id")
+        case_set = Cases.objects.filter(community_id=1).select_related('victim_id')
+        genders_to_counts = {
+            0: 0,
+            1: 0,
+            2: 0,
+            3: 0
+        }
+        total_count = 0
```

Comment:
This can be something like:
        genders_to_counts = {
            'Female': 0,
            'Male': 0,
            'Other': 0,
            'Total Count': 0
        }

Using the method: victim.get_gender_display() in the loop below will get the corresponding string from the stored int.


the dictionary count is no longer needed and genders_to_counts can be returned instead.